api/control/v1alpha1/types.go

```diff
@@ -403,7 +411,9 @@ const (
 type Expression struct {
 	Op   ExpressionOperator `json:"op"`
 	Path string             `json:"path,omitempty"`
-	Args []Expression       `json:"args,omitempty"`
+	// +kubebuilder:pruning:PreserveUnknownFields
+	// +kubebuilder:validation:Schemaless
```

Comment:
Why mark this as schemalesss? (https://book.kubebuilder.io/reference/markers/crd-validation)